docs/source/normalization.rst

```diff
@@ -20,15 +20,223 @@ An artifact is any field or subset of data within a record that bears meaning be
+        "calendarTime": "string",
+        "columns": {},
+        "counter": "integer",
+        "decorations": {},
+        "epoch": "integer",
+        "hostIdentifier": "string",
+        "log_type": "string",
+        "name": "string",
+        "unixTime": "integer",
+        "logNumericsAsNumbers": "string",
+        "numerics": "string"
+      },
+      "parser": "json",
+      "configuration": {
+        "optional_top_level_keys": [
+          "counter",
+          "decorations",
+          "epoch",
+          "log_type",
+          "logNumericsAsNumbers",
+          "numerics"
+        ],
+        "normalization": {
+          "command": [
+            "cmdline",
+            "command"
+          ],
+          "ip_address": [
+            {
+              "fields": [
+                "host",
+                "source",
+                "local_address",
+                "address"
+              ],
+              "function": "Source IP addresses"
+            },
+            {
+              "fields": [
+                "destination",
+                "remote_address",
+                "gateway"
+              ],
+              "function": "Destination IP addresses"
+            }
+          ],
+          "user_identity": [
+            "username",
+            "user"
+          ]
+        }
+      }
+    }
 
 Deployment
 ==========
 
-Stay tuned.
+* Artifact Extractor will only work if firehose and historical search enabled in ``conf/global.json``
+
+  .. code-block::
+
+    "infrastructure": {
+      ...
+      "firehose": {
+        "use_prefix": true,
+        "buffer_interval": 60,
+        "buffer_size": 128,
+        "enabled": true,
+        "enabled_logs": {
+          "cloudwatch": {},
+          "osquery": {}
+        }
+      }
+      ...
+    }
+
+* Enable Artifact Extractor feature in ``conf/lambda.json``
+
+  .. code-block::
+
+    "artifact_extractor_config": {
+      "concurrency_limit": 10,
+      "enabled": true,
+      ...
+    },
+
+* Use StreamAlert cli to deploy Artifact Extractor lambda function and new resources
+
+  The deployment will add following resources.
+
+  * A new Lambda function
+  * A new Glue catalog table ``artifacts`` for Historical Search via Athena
+  * A new Firehose to deliver artifacts to S3 bucket
+  * Update existing Firehoses to allow to invoke Artifact Extractor lambda if it is enabled on the Firehoses
+  * New permissions, metrics and alarms.
+
+  .. code-block:: bash
+
+    python manage.py deploy --function artifact_extractor
+
+* If normalization configuration changed in ``conf/schemas/*.json``, make sure deploy classifier as well
+
+  .. code-block:: bash
+
+    python manage.py deploy --function classifier
+
+Artifacts
+=========
+
+Artifacts will be searching via Athena ``artifacts`` table. During the test in staging environment, two fake ``cloudwatch:events`` were sent to a Kinesis data stream.
+
+Those two fake events were searchable in ``cloudwatch_events`` table.
```

Comment:
what are these references to "fake" events and "staging" environment in the docs??? please remove or update as necessary